src/backend/README.md

```diff
@@ -0,0 +1,33 @@
+# How to Set Up
+
+## Set Up with Docker 
+If you already have Docker and docker-compose installed on your local environment, you can directly use the following command to build and start the project. (Make sure your Docker Daemon is running)
+```Shell
+chmod +x run.sh
```

Comment:
What are the benefits of using shell commands over executing docker-compose directly?